buddyboss/buddyboss-platform
# Fix - Line break is not working in the Paragraph Text Field

### All Submissions:


Fixes #2774 

### How to test the changes in this Pull Request:

1. Go to member's profile tab
2. Edit profile
3. Select the biography tab
4. Add content with line break
5. Save changes
6. Check the issue in the member's biography content

### Proof Screenshots or Video
https://prnt.sc/1qjbha7

### Other information:
